### DISCUSSION TITLE
关于日语、漫画等orc识别

### DISCUSSION THREAD

**Discussion by panda114514667** [STranslate/STranslate#186] [Ideas]
我在使用软件过程中发现paddleORC在识别漫画日文时效果经常不理想，于是我在github上找到了另外一个项目[https://github.com/kha-white/manga-ocr](https://github.com/kha-white/manga-ocr)，从这个项目介绍中他的数据集对识别日语漫画有着专门攻略。但是该项目作者的终端方案不如ST高效，所以能否通过数据包加载的方式使用上manga-orc的日文漫画数据包

**Comment by ZGGSONG** [comment]
不好意思，该项目API方案不是很完善，我暂时没有这个计划去实现该方案
